run_tests.py

```diff
@@ -14,17 +15,9 @@
 # limitations under the License.
 """Script to run the tests."""
 
-import os
-import sys
-import unittest
+import subprocess
 
 if __name__ == '__main__':
-  base_dir = os.path.dirname(os.path.abspath(__file__))
-  # Change PYTHONPATH to include full path to turbinia.
-  sys.path.insert(0, base_dir)
-
-  test_suite = unittest.TestLoader().discover(
-      os.path.join(base_dir, 'tests'), pattern='*.py')
-  test_results = unittest.TextTestRunner(verbosity=2).run(test_suite)
-  if not test_results.wasSuccessful():
-    sys.exit(1)
+  subprocess.check_call([
```

Comment:
I think I had just copied what Plaso was doing for this.  Is this what timesketch is doing now as well?

Reply:
I'm using nosetests in Timesketch the same way as this PR. Way easier to run the tests on Travis and it goes faster as well. Having the test files side by side is nicer IMO as it is very clear what have tests and what doesnt have. Let me know what you think.